detection-rules/attachment_invoice_fake_customer_service_freemail_sender.yml

```diff
@@ -1,4 +1,4 @@
-name: "Attachment invoice or receipt from freemail sender with fraudulent customer service number"
+name: "Attachment invoice or receipt from freemail sender with customer service number"
```

Comment:
Should this be "Attachment: Invoice or receipt from freemail sender with customer service number" ?